app-text/mupdf/mupdf-1.19.1.recipe

```diff
@@ -0,0 +1,127 @@
+SUMMARY="A lightweight XPS and PDF rendering library"
+DESCRIPTION="The renderer in MuPDF is tailored for high quality anti-aliased \
+graphics. It renders text with metrics and spacing accurate to within \
+fractions of a pixel for the highest fidelity in reproducing the look of a \
+printed page on screen.
+
+MuPDF is also small, fast, and yet complete. It supports PDF 1.7 with \
+transparency, encryption, hyperlinks, annotations, searching and more. It also \
+reads XPS/OpenXPS documents and CBZ (Comic Book archive) files."
+HOMEPAGE="https://www.mupdf.com/"
+COPYRIGHT="2006-2022 Artifex Software, Inc"
+LICENSE="AGPL-3.0"
+REVISION="1"
+SOURCE_URI="https://mupdf.com/downloads/archive/mupdf-$portVersion-source.tar.xz"
+CHECKSUM_SHA256="b5eac663fe74f33c430eda342f655cf41fa73d71610f0884768a856a82e3803e"
+SOURCE_DIR="mupdf-$portVersion-source"
+PATCHES="mupdf-$portVersion.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	mupdf$secondaryArchSuffix = $portVersion
+	lib:libmupdf$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libharfbuzz$secondaryArchSuffix
+	lib:libjbig2dec$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libopenjp2$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	mupdf${secondaryArchSuffix}_devel = $portVersion
+	devel:libmupdf$secondaryArchSuffix
+	"
+REQUIRES_devel="
+	mupdf$secondaryArchSuffix == $portVersion base
+	"
+
+SUMMARY_tools="Tools to convert, examine and rasterise PDF files"
+PROVIDES_tools="
+	mupdf${secondaryArchSuffix}_tools = $portVersion
+	cmd:muraster$commandSuffix
+	cmd:mutool$commandSuffix
+	"
+REQUIRES_tools="
+	haiku$secondaryArchSuffix
+	mupdf$secondaryArchSuffix == $portVersion base
+	lib:libcrypto$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libharfbuzz$secondaryArchSuffix
+	lib:libjbig2dec$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libopenjp2$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
```

Comment:
libcurl and libglib_2.0 not needed here:
```diff
	lib:libcrypto$secondaryArchSuffix
-	lib:libcurl$secondaryArchSuffix
	lib:libfreetype$secondaryArchSuffix
-	lib:libglib_2.0$secondaryArchSuffix
	lib:libharfbuzz$secondaryArchSuffix
	lib:libjbig2dec$secondaryArchSuffix
	lib:libjpeg$secondaryArchSuffix
	lib:libopenjp2$secondaryArchSuffix
	lib:libz$secondaryArchSuffix
```

Reply:
Removed. Probably unrelated but the build failed because it couldn't find python3 so I have added that to the BUILD_PREREQUIRES

Reply:
Jikes, you removed them in all places (also in the patchset)? These 2 are not needed for the tools package, for the base and develop package you can leave them in (and also the search for them in the patch)

Reply:
Okay I'll revert this and then remove them only for the tools package

Reply:
I'm curious as to how you could tell they weren't needed here? Sorted anyway